.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jsf9k @mcdonnnj
```

Comment:
This saddens me.

Reply:
I have been doing it for at least a year now.  Given your rare appearances on PRs I thought it made sense.  I am sorry you are sad.